README.md

```diff
@@ -254,10 +255,27 @@ variable to `event`. For more details on this signature type, check out the Goog
 documentation on
 [background functions](https://cloud.google.com/functions/docs/writing/background#cloud_pubsub_example).
 
-# CloudEvents Support
+# Enable CloudEvents
 
-The Functions Framework will soon support [CloudEvents](http://cloudevents.io).
-Please contact the repository owners if you're intersted in trying this feature.
+The Functions Framework can unmarshall incoming [CloudEvents](http://cloudevents.io)
+payloads to a `cloudevent` object. This will be passed as arguments to your function when it
+receives a request. Note that your function must use the cloudevent-style function
+signature:
+
+```php
+function helloCloudEvents(CloudEvent $cloudevent)
```

Comment:
nit: Since this shows a full function, the use statement for `CloudEvent` should be included.

Reply:
SG. Fixed.

Reply:
SG. Fixed.